### ISSUE TITLE
USSD Code is sent as number

### ISSUE THREAD

**Issue by tusch001** [SimpleMobileTools/Simple-Dialer#47]
With my prepaid sim card USSD Code `*100#` returns a menu where I can get info for my account.

This code is saved as a contact. With the old way over the contact app I could dial this USSD code. With the new dialer the (phone) number `100` is sent instead and therefore my provider tells me that this is not a valid phone number.

**Comment by renarena** [comment]
i have the same problem with *31#
I think this is an error, not a feature request.
I am not able to call someone with a not hidden number, if hidden is default on my phone.
Calling from contact app works as expected.

**Comment by mkr4** [comment]
Same here.
My provider even made paid service on that number. :)

**Comment by l3u** [comment]
I also hit this issue trying out the Simple Dialer. Same here, I can "call" `*100#` to see my prepaid balance. I also have an address book entry for this "contact", and "calling" it from the Simple Contacts app executes the USSD code. Doing the same from the Simple Dialer actually starts a phone call to `100`.

Imho, this is not a feature request, but a bug – and it would really be cool if this was fixed.

Thanks for providing those great Google stock app alternatives :-)

**Comment by FestplattenSchnitzel** [comment]
With v5.7.0 (dialler) and v6.15.0 (contacts), both from F-Droid, it works fine for me.

**Comment by anedroid** [comment]
Version 5.11.5, bug still present. If I add a homescreen shortcut to any USSD numer - for example *107# for checking how much transfer is still available - simple dialer calls 107 instead of *107#. Please, fix this, because Simple Dialer is the best free software replacement to Google dialer at the moment.

**Comment by szero** [comment]
Yup, if you store your USSD code through Simple Contacts app and try to dial it through this app, it strips * and # chars from the number, kind of annoying, you have to type in stuff manually.
